### DISCUSSION TITLE
Encriptar os dados

### DISCUSSION THREAD

**Discussion by renatoianhez** [sepinf-inc/IPED#2359] [Q&A]
Olá! Trabalho com perícia em celulares (PCMG) e temos tido ótimos resultados com o IPED. 
Minha questão é: quando fazemos o relatório de extrações via Cellebrite, usamos uma função do programa, no momento da criação do relatório, de encriptação dos dados e o uso de uma senha para abrir o relatório. Há alguma função semelhante no IPED ao se abrir o IPED-SearchApp? Se não existe,  há a previsão de ser incluída?
Obrigado!

**Comment by lfcnassif** [comment]
Hello. Unfortunately there is no such feature and it is not planned to be included today. You can encrypt your cases using zip or 7zip encryption.
